project.clj

```diff
@@ -37,7 +37,7 @@
                  [re-frame "0.10.9"]
                  [reagent "0.7.0"]
                  [garden "1.3.2"]
-                 [org.apache.pdfbox/pdfbox "2.1.0-20170324.170253-831"]
+                 [org.apache.pdfbox/pdfbox "2.0.25"]
```

Comment:
You didn't test with this version - I have and it doesn't produce PDF's.
I have a pull request coming to fix PdfBox so you can remove it from this pull request.

Reply:
https://github.com/datdamnzotz/orcpub/commit/f02d84e775f05c8ad637163f2ddd9de2a869a101

Reply:
You're right; I overlooked your fix.  Sorry 'bout that.